src/test/kotlin/lotto/ProfitTest.kt

```diff
@@ -0,0 +1,21 @@
+package lotto
+
+import lotto.domain.LottoPrize
+import lotto.domain.Money
+import lotto.domain.PositiveNumber
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+internal class ProfitTest {
+
+    @Test
+    fun `당첨된 로또가 없으면 수익률은 0이다`() {
+        val prizeRankCount = mapOf(Pair(LottoPrize.WHACK, PositiveNumber(1)))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val prizeRankCount = mapOf(LottoPrize.WHACK to PositiveNumber(1))
```

Reply:
`to` 를 사용하니 더 간결해지네요 😁 4ce2d7d